### PR TITLE
TT2-1693: Changing all test emails have the @example.com domain

### PR DESCRIPTION
**Pull request by dannymadden-gds**
- **Ticket Number**: #TT2-1693 :ticket:

Description:

Changing test email domains to @example.com and where appropriate, leaving domains at gov.uk and changing what preceded it to 'example'

### CODE REVIEW

**Review by andrewwalter-gds — DISMISSED** [review]

File: tests/e2e-tests/.env.template [hunk 1]
```diff
@@ -1,9 +1,9 @@
 process.env.AUDIT_BUCKET_NAME = 'audit-build-message-batch-txma2'
 process.env.NOTIFY_API_KEY = 'NOTIFY_API_KEY'
 process.env.TEST_DATA_BUCKET_NAME = 'txma-audit-dev-tools-build-test-data-bucket'
-process.env.ZENDESK_ADMIN_EMAIL = 'txma-team2-ticf-admin-dev@test.gov.uk'
-process.env.ZENDESK_AGENT_EMAIL = 'txma-team2-ticf-approver-dev@test.gov.uk'
+process.env.ZENDESK_ADMIN_EMAIL = 'txma-team2-ticf-admin-dev@example.com'
```
Comment (richa-misra) [inline tests/e2e-tests/.env.template:4]: these ok to have their domain changed from gov.uk to .com?

**Review by richa-misra — APPROVED** [review]

File: tests/e2e-tests/.env.template (hunk 1 above)
Comment (dannymadden-gds) [reply tests/e2e-tests/.env.template]: have updated quite a few of the test emails back to gov.uk - more appropriate

**Review by andrewwalter-gds — DISMISSED** [review]

File: tests/e2e-tests/.env.template (hunk 1 above)
Comment (zaydgds) [reply tests/e2e-tests/.env.template]: can we avoid changing these as these are part of the tests

**Review by zaydgds — CHANGES_REQUESTED** [review]
i'd avoid changing this repo in general as this relies on some test data in s3 which will also need changing.

i havent quite figured out how the multiple sets of test data across the various buckets are linked to what tests so i'd avoid changing this repo

i'd avoid changing


- `test/*`
- `scripts/setupDevStackSecrets.ts`

**Review by zaydgds — APPROVED** [review]

**Review by andrewwalter-gds — APPROVED** [review]

**Pull request merged**